Documentation/ApiOverview/GlobalValues/Typo3ConfVars/SYS.rst

```diff
@@ -0,0 +1,811 @@
+===================================================
+
+.. confval:: encryptionKey
+
+   :type: text
+   :Default: ''
+
+   This is a "salt" used for various kinds of encryption, CRC checksums and
+   validations. You can enter any rubbish string here but try to keep it
+   secret. You should notice that a change to this value might invalidate
+   temporary information, URLs etc. At least, clear all cache if you change
+   this so any such information can be rebuilt with the new key.
+
+.. index::
+   TYPO3_CONF_VARS SYS; cookieDomain
+.. _typo3ConfVars_sys_cookieDomain:
+
+$GLOBALS['TYPO3_CONF_VARS']['SYS']['cookieDomain']
+==================================================
+
+.. confval:: cookieDomain
+
+   :type: text
+   :Default: ''
+
+   Restricts the domain name for FE and BE session cookies. When setting the
+   value to ".domain.com" (replace domain.com with your domain!), login
+   sessions will be shared across subdomains. Alternatively, if you have more
+   than one domain with sub-domains, you can set the value to a regular
+   expression to match against the domain of the HTTP request.
+
+   The result of the match is used as the domain for the cookie. for example :
+   php:`/\.(example1|example2)\.com$/` or :php:`/\.(example1\.com)|(example2\.net)$/`.
+   Separate domains for FE and BE can be set using
+   :ref:`$TYPO3_CONF_VARS[FE][cookieDomain]<typo3ConfVars_fe_cookieDomain>` and
+   :ref:`$TYPO3_CONF_VARS[BE][cookieDomain]<typo3ConfVars_be_cookieDomain>`
+   respectively.
+
+.. index::
+   TYPO3_CONF_VARS SYS; trustedHostsPattern
+.. _typo3ConfVars_sys_trustedHostsPattern:
+
+$GLOBALS['TYPO3_CONF_VARS']['SYS']['trustedHostsPattern']
+=========================================================
+
+.. confval:: trustedHostsPattern
+
+   :type: text
+   :Default: 'SERVER_NAME'
+
+   Regular expression pattern that matches all allowed hostnames (including
+   their ports) of this TYPO3 installation, or the string :php:`SERVER_NAME`
+   (default).
+
+   The default value :php:`SERVER_NAME` checks if the HTTP Host header equals
+   the SERVER_NAME and SERVER_PORT. This is secure in correctly configured
+   hosting environments and does not need further configuration. If you cannot
+   change your hosting environment, you can enter a regular expression here.
+
+   Examples:
+
+   :php:`.*\.domain\.com` matches all hosts that end with
+   :file:`.domain.com` with all corresponding subdomains.
+
+   :php:`(.*\.domain|.*\.otherdomain)\.com` matches all hostnames with
+   subdomains from :file:`.domain.com` and :file:`.otherdomain.com`.
+
+   Be aware that HTTP Host header may also contain a port. If your installation
+
+   runs on a specific port, you need to explicitly allow this in your pattern,
+
+   for example :php:`www\.domain\.com:88` allows only :file:`www.domain.com:88`,
+   **not** :file:`www.domain.com`. To disable this check completely
+
+   (not recommended because it is **insecure**) you can use :php:`.*` as pattern.
```

Comment:
```suggestion
   Be aware that HTTP Host header may also contain a port. If your installation
   runs on a specific port, you need to explicitly allow this in your pattern,
   for example :php:`www\.domain\.com:88` allows only :file:`www.domain.com:88`,
   **not** :file:`www.domain.com`. To disable this check completely
   (not recommended because it is **insecure**) you can use :php:`.*` as pattern.
```